lang_python.adoc

```diff
@@ -33,68 +33,94 @@ Runtime library API version must match compiler version that you're using
 (i.e. if you use ksc v0.8, then you must use compatible v0.8 runtime).
 
 [[add-runtime-release]]
-==== Using release version
+==== Using release version (NOT recommended)
```

Comment:
Can you elaborate, how exactly is using release is "not recommended"?

Reply:
I personally don't recommend since they are outdated most of the time.